### PR TITLE
Fix #271: Add QR Code support

### LINKED ISSUE #271: Plugin: QR Code
I will create a plugin that allows someone to add a QRCode to their website.

```
{qrcode value="http://melloware.com" foreground="blue" background="white"}
```

A lot of restaurant static websites have links to their "menu" etc and want to have a QRCode that people can scan.

### PR DESCRIPTION
**Pull request by melloware**
Fix #271: Add QR Code support

image

### CODE REVIEW

**Comment by mcruzdev** [comment]
`quarkus-doc-plugin.adoc:155`, see [here](https://github.com/quarkiverse/quarkus-roq/actions/runs/11842699209/job/33001912848?pr=275#step:5:5531).

File: docs/modules/ROOT/pages/quarkus-roq-plugins.adoc [hunk 1]
```diff
@@ -129,4 +129,27 @@ To install it add:
...
+    <groupId>io.quarkiverse.roq</groupId>
+    <artifactId>quarkus-roq-plugin-qrcode</artifactId>
+    <version>{project-version}</version>
+</dependency>
+----
+
+Then create a template and add the `{#qrcode ...}` tag to it and style and size it as you want.
+
+[source,yaml]
+
```
Comment (mathecruz) [inline docs/modules/ROOT/pages/quarkus-roq-plugins.adoc]:
```suggestion
----
```

**Comment by mcruzdev** [comment]
This one is a greate feature @melloware!

**Comment by melloware** [comment]
Fixed!

**Comment by melloware** [comment]
Thanks to @FroMage who i stole some of this code from Renarde!

**Review by mcruzdev — APPROVED** [review]

**Pull request merged**